tools/download_from_gcs.py

```diff
@@ -54,27 +51,23 @@ def ExtractSha1(filename):
 
 def _DownloadFromGcsAndCheckSha1(bucket, sha1):
   url = f'{_BASE_GCS_URL}/{bucket}/{sha1}'
-  context = create_default_context()
-
   try:
-    res = urllib.urlopen(
-        url, context=context) if context else urllib.urlopen(url)
-  except urllib.URLError:
-    from ssl import _create_unverified_context  # pylint:disable=import-outside-toplevel
-    context = _create_unverified_context()
-    res = urllib.urlopen(
-        url, context=context) if context else urllib.urlopen(url)
+    # pylint: disable=consider-using-with
+    res = urllib.request.urlopen(url, context=create_default_context())
+  except urllib.error.URLError:
+    # Retry without ssl verification.
+    # pylint: disable=consider-using-with
+    res = urllib.request.urlopen(url, context=_create_unverified_context())
```

Comment:
Do you remember/know why we have to do this?

Reply:
I looked at this and was hoping I could remove it but in b/185935607 @arjungm encountered errors on some windows builds.

Reply:
I think the error might have been py2 specific, maybe we could try getting rid of this? (Don't have to here)

Reply:
Sgtm